pkg/katautils/config.go

```diff
@@ -123,6 +123,8 @@ type hypervisor struct {
 	DisableImageNvdimm      bool     `toml:"disable_image_nvdimm"`
 	HotplugVFIOOnRootBus    bool     `toml:"hotplug_vfio_on_root_bus"`
 	DisableVhostNet         bool     `toml:"disable_vhost_net"`
+	EnableVhostUserStore    bool     `toml:"enable_vhost_user_store"`
```

Comment:
Do we need this flag? Would we want to disable this feature for not supporting vhost-user devices?
The reason I bring this up, is that we already have a bloat of supported configs and do not want to add to these unless really required.

Reply:
Eh... Yes, having too many configs is really a problem.
At the time, I added this config option about the enablement. I thought this feature will occupy some Linux reserved block device numbers. But we didn't know whether these numbers are already utilized for other purpose in user's circumstance. If some user is going to use vhost-user storage devices, he should be aware that there is no conflict about that by the enablement config option.
So this config is still there.

Reply:
This is fine with me. It may be useful to have the reserved block numbers configurable in the future. But we can get to that later if we see the need.

Reply:
:)